packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -282,7 +302,16 @@ private void sendInitialized() {
         }
         event.put("width", width);
         event.put("height", height);
+
+        // Rotating the video with ExoPlayer does not seem to be possible with a Surface,
+        // so inform the Flutter code that the widget needs to be rotated to prevent
+        // upside-down playback for videos with rotationDegrees of 180 (other orientations work
+        // correctly without correction).
+        if (rotationDegrees == 180) {
```

Comment:
is there any other use case  where `rotationDegrees` is useful?  I was wondering if sending the value as `rotationDegrees` could be beneficial.  Other than that, this change LGTM

Reply:
`rotationCorrection` is the name it has in the (platform-interface-level) `VideoEvent` that this populates, so for now it's the clearest name for the event.

Now that channels are per-platform-implementation package, if we have a second use for this within Android in the future we can trivially change it at that point.

Reply:
Also see https://github.com/flutter/plugins/pull/3820#discussion_r715300418 where we discussed this previously.

I think it would be nice to eventually pass along the rotationDegrees info, but we'd probably want to provide that on all platforms if possible (not just Android).